packages/client/scripts/postinstall.js

```diff
@@ -197,30 +194,49 @@ function run(cmd, params, cwd = process.cwd()) {
   })
 }
 
-async function ensureEmptyDotPrisma() {
+/**
+ * Copies our default "throw" files into the default generation folder. These
+ * files are dummy and informative because they just throw an error to let the
+ * user know that they have forgotten to run `prisma generate`.
```

Comment:
- Maybe mention `node_modules/.prisma/client` as the default generateion folder
- Technically this does not have to mean that they forgot, but also that they might not have a schema file yet. Maye reword that it just means they or package installation did not run `prisma generate` successfully yet to create a generated Client in that location

Reply:
I will update this in the finalization PR.